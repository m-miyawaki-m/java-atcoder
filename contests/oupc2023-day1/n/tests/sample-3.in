7 20
3 1 4
2 -2 7
4 1 1
1 -3 8
9 2 4
6 3 1
4 -2 5
6
8
0
3
10
17
2
