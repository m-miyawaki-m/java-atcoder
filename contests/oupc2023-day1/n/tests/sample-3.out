13
12
13
14
15
12
